app/controllers/mobile.js

```diff
@@ -41,4 +43,26 @@ export default class MobileController extends Controller {
       // will cancel the login
     }
   }
+
+  @action async buttonClicked() {
```

Comment:
The problem is that the verifyAuth() function is not called asynchronously. This means that the buttonClicked() function will block until the verifyAuth() function has finished executing. This can cause problems if the verifyAuth() function takes a long time to execute, or if it throws an error.

Reply:
please check recent commit